modules/integration/tests-integration/tests-backend/src/test/java/org/wso2/am/integration/tests/other/SameVersionAPITestCase.java

```diff
@@ -22,117 +22,110 @@
 
 import org.apache.commons.logging.Log;
 import org.apache.commons.logging.LogFactory;
-import static org.testng.Assert.assertEquals;
-import static org.testng.Assert.assertTrue;
 
 import org.testng.annotations.*;
-import org.wso2.am.admin.clients.webapp.WebAppAdminClient;
+import org.wso2.am.integration.clients.publisher.api.ApiException;
+import org.wso2.am.integration.test.impl.RestAPIPublisherImpl;
 import org.wso2.am.integration.test.utils.base.APIMIntegrationBaseTest;
 import org.wso2.am.integration.test.utils.base.APIMIntegrationConstants;
-import org.wso2.am.integration.test.utils.bean.APICreationRequestBean;
-import org.wso2.am.integration.test.utils.bean.APIResourceBean;
-import org.wso2.am.integration.test.utils.clients.APIPublisherRestClient;
-import org.wso2.am.integration.test.utils.generic.TestConfigurationProvider;
-import org.wso2.am.integration.test.utils.webapp.WebAppDeploymentUtil;
+import org.wso2.am.integration.test.utils.bean.APIRequest;
 import org.wso2.carbon.automation.engine.context.TestUserMode;
-import org.wso2.carbon.automation.engine.frameworkutils.FrameworkPathUtil;
-import org.wso2.carbon.automation.test.utils.common.FileManager;
 import org.wso2.carbon.automation.test.utils.http.client.HttpResponse;
 
 import javax.ws.rs.core.Response;
-import java.io.File;
 import java.net.URL;
-import java.util.ArrayList;
-import java.util.List;
-
-public class SameVersionAPITestCase extends APIMIntegrationBaseTest{
-
-    private static final Log log= LogFactory.getLog(SameVersionAPITestCase.class);
-    private APIPublisherRestClient apiPublisher;
-    private static final String API_NAME="SameVersionAPITest";
-    private static final String API_CONTEXT="SameVersionAPI";
-    private String version="1.0.0";
-    private String newVersion="1.0.0";
-    private String TAGS="testtag1, testtag2";
+
+import static org.testng.Assert.*;
```

Comment:
Shall we import only the required rather than importing the entire package?

Reply:
fixed